pabi_procurement/models/purchase_requisition.py

```diff
@@ -38,10 +38,20 @@ class PurchaseRequisition(models.Model):
         'requisition_id',
         'Attach Files',
     )
+    amount_untaxed = fields.Float(
+        'Untaxed Amount',
+        readonly=True,
+        default=0.0
+    )
+    amount_tax = fields.Float(
+        'Taxes',
+        readonly=True,
+        default=0.0
+    )
     amount_total = fields.Float(
         'Total',
         readonly=True,
-        default=0.0,
+        default=0.0
```

Comment:
For python, the right way is to have "," at the end of each line. In this case, use

  default=0.0,
)

Reference: https://mobileapp.nstda.or.th/redmine/projects/new-pabi/wiki/Coding_Standard#Patterns